cli/tsc/00_typescript.js

```diff
@@ -7375,7 +7375,7 @@ var ts;
                     close: function () { return _fs.unwatchFile(fileName, fileChanged); }
                 };
                 function fileChanged(curr, prev) {
-                    // previous event kind check is to ensure we recongnize the file as previously also missing when it is restored or renamed twice (that is it disappears and reappears)
+                    // previous event kind check is to ensure we recognize the file as previously also missing when it is restored or renamed twice (that is it disappears and reappears)
```

Comment:
Please do not modify this file, it is obtained upstream.  If you want to fix typos here, please consider contributing to microsoft/TypeScript and those will flow through to here.

Reply:
Reverted this change